L/Libglvnd/build_tarballs.jl

```diff
@@ -50,5 +54,14 @@ dependencies = [
     BuildDependency("Xorg_xorgproto_jll"),
 ]
 
+init_block = raw"""
+    if Sys.islinux()
+        get!(ENV, "__EGL_VENDOR_LIBRARY_DIRS", "/usr/share/glvnd/egl_vendor.d")
+    elseif Sys.isfreebsd()
+        get!(ENV, "__EGL_VENDOR_LIBRARY_DIRS", "/usr/local/share/glvnd/egl_vendor.d")
```

Comment:
Isn't there anything better than hardcoding absolute paths?

Reply:
I'm open to suggestions, but I haven't found a lot of useful information other than the linked icd_enumeration.md file about how the library should be configured to find various GL implementations. (It took quite a while of staring at `strace` outputs to get as far as knowing it's libglvnd which was the key link in the chain to getting GLFW to work on Wayland.)

At some level, I think there has to be an assumed absolute path somewhere — the point of Libglvnd is to use the `egl_vendor.d` directory to avoid hard-coding variations for Intel, AMD, NVIDIA, etc GL libraries. And more to the point, there's already an incorrect hard-coded absolute path built-in to the artifact:
```julia-repl
julia> using Libglvnd_jll

julia> # embedded string within the Libglvnd_jll artifact

julia> foreach(print, split(read(`strings $(Libglvnd_jll.libEGL)`, String), '\n') |> filter(contains("egl_vendor.d")))
/workspace/destdir/etc/glvnd/egl_vendor.d:/workspace/destdir/share/glvnd/egl_vendor.d

julia> # embedded string within my local system's libEGL.so

julia> foreach(print, split(read(`strings /usr/lib/libEGL.so`, String), '\n') |> filter(contains("egl_vendor.d")))
/etc/glvnd/egl_vendor.d:/usr/share/glvnd/egl_vendor.d
```

Rather than trying to split based on OS, though, I could simplify the statement to just
```julia
         get!(ENV, "__EGL_VENDOR_LIBRARY_DIRS", "/etc/glvnd/egl_vendor.d:/usr/share/glvnd/egl_vendor.d:/usr/local/share/glvnd/egl_vendor.d")
```
and cover all paths I've been able to find in one go.

Reply:
Maybe as a useful comparison, I went and dug through the OpenCL loader code and found that it hard-codes a path during compile to `/etc/OpenCL/vendors/`: https://github.com/KhronosGroup/OpenCL-ICD-Loader/blob/c2cbdb33605ef9cfa02aa2a77e9cd7306c57e32a/loader/icd_platform.h#L30

The difference here is that the compile-time path makes use of the build `${prefix}` whereas the OpenCL loader ignores that and just hard-codes the normal system path.

If it'd be preferable to patch the build to similarly hard-code the destination rather than doing a runtime setup of the environment variable, I can probably do that this weekend.

Reply:
I've switched to patching the build to ignore the build `$prefix` in setting the default `__EGL_VENDOR_LIBRARY_DIRS` path, as a way to avoid the `__init__()` method from being needed.